app/build.gradle

```diff
@@ -605,8 +605,14 @@ dependencies {
 }
 
 protobuf {
+    // Mac M1 workaround until we can bump the version. Dependent on A-S.
+    // See https://github.com/mozilla-mobile/fenix/issues/22321
```

Comment:
Thanks for adding a link to the issue so we can know when to remove this workaround!